### DISCUSSION TITLE
DETR모델로 학습

### DISCUSSION THREAD

**Discussion by kmouleejunhyuk** [boostcampaitech2/object-detection-level2-cv-06#12] [ideas]
- 논문 링크: [https://arxiv.org/pdf/2005.12872.pdf](https://arxiv.org/pdf/2005.12872.pdf)
- DETR은 NMS를 사용하지 않는 것으로 보이기에 겹쳐진 box문제를 해결할 수 있을 것으로 생각
    - With its set-based loss, DETR does not need NMS by design
    
    ![image](https://user-images.githubusercontent.com/49234207/135575811-e75ba643-0ced-4ff9-b789-3361da50cb09.png)
    
- 실험결과
    #24

**Comment by ppskj178** [comment]
실험결과를 모아보려면
결과는 show-and-tell 카테고리에 별도로 작성한 한 후에 링크를 거는게 좋을 것같습니다.

**Comment by jaehwan-AI** [comment]
thr를 바꿔가면서 결과를 볼 수 있을까요?

**Reply by kmouleejunhyuk** [reply]
@jaehwan-AI   넵 #24 에 업로드했습니다
